src/components/EditCard.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+    Drawer,
+    DrawerBody,
+    DrawerFooter,
+    DrawerHeader,
+    DrawerOverlay,
+    DrawerContent,
+    DrawerCloseButton,
+    Button,
+    useDisclosure,
+    Input,
+    IconButton,
+} from "@chakra-ui/react";
+import { EditIcon } from "@chakra-ui/icons";
+import { useState } from "react";
+import axios from "axios";
+import EditCardProps from "../interfaces/EditCardProps";
+
+function EditCard({
```

Comment:
This is the same component as AddBoard, can reuse one component with a different onClick handler for the save button